p2p/chain.py

```diff
@@ -203,6 +202,10 @@ def register_peer(self, peer: BasePeer) -> None:
                 self.logger.warn("Timeout waiting for header batch from %s, aborting sync", peer)
                 await peer.disconnect(DisconnectReason.timeout)
                 break
+            except ValidationError:
+                self.logger.warn("Invalid header response sent by peer.")
```

Comment:
It'd be good to include the original exception in the log, no? I'd also mention we'll disconnect from the peer because of that